lib/ansible/modules/windows/win_psmodule.ps1

```diff
@@ -72,7 +89,7 @@ Function Remove-Repository{
     [bool]$CheckMode
     )
 
-    $Repo = (Get-PSRepository).SourceLocation
+    $Repo = (Get-PSRepository).Name
```

Comment:
What is your reasoning behind this, if it is a bug we should change it in a separate PR and backport it.

Reply:
Yes, it is a bug.
There is already PR #45566 that fixes it. 
When i created my PR, #45566 had not been created yet, so i had fix it to pass tests.